lib/gc/Transforms/DeepTileContractionOp.cpp

```diff
@@ -947,18 +957,19 @@ struct DeepTileMatmul : public OpInterfaceRewritePattern<linalg::LinalgOp> {
 
     // Step 1. Split matmul(bf16xbf16->bf16) to matmul(bf16xbf16->f32) +
     // cast(f32->bf16) if K slicing is needed
-    MatmulConfig cfg =
-        MatmulConfigAnalysis(originOp.getOperation()).getConfig();
-    linalgOp = *linalg::generalizeNamedOp(rewriter, linalgOp);
+    MatmulConfigAnalysis cfgAnalysis =
+        MatmulConfigAnalysis(originOp.getOperation());
+    cfgAnalysis.setAllowUndivisibleInnerBlock(false);
```

Comment:
If we by default not allow indivisible inner block, will we still receive empty valid config for DLRM top/bot MLP workload (where K can be 13 or 479).